host_guest/GDCC_and_guests/README.md

```diff
@@ -18,21 +18,21 @@ In the former they are remote from the non-polar pocket, whilst in the latter th
 ![](../../images/GDCCs.jpg)
 
 The Gibb group will examine the binding of eight guests: four negatively charged, and four positively charged.
-As previously, all binding constants will be determined in sodium phosphate buffer at pH 11.5.
+As previously, all binding constants will be determined in (10 mM) sodium phosphate buffer, here at pH 11.7.
 The guests have been selected to partially overlap with previous determinations for binding to OA (first row of guests; known ITC values to be added to this repository later).
 These six examples will act as a reference point for the other determinations.
 There will be two new determinations for binding to OA (second row of guests), as well as the eight new determinations using new host exo-OA.
 Each of the ten new determinations will be carried out in triplicate using ITC, and where necessary, verified by NMR.
 
 ![](../../images/GDCC_guests.jpg)
 
-Given the identity of these guests, it is likely we will be able to have *some* overlap between the GDCC guests and those used in the modified cyclodextrin (Gilson lab) and acyclic CB (Isaacs lab) portions of the challenge.
+Given the identity of these guests, we will be able to have *some* overlap between the GDCC guests and those used in the modified cyclodextrin (Gilson lab) and acyclic CB (Isaacs lab) portions of the challenge.
 
 ### Additional technical details
 
-Buffer conditions are expected to be 10 mM sodium phosphate, but this will be confirmed as data collection begins. The plan is to gather data in triplicate with fresh solutions of host and guest on each occasion.  Also, all hosts samples are probed by one of either two ways to determine the waters of hydration in each sample.  This can be as high so this analysis is a requirement to avoid bad data.
+Buffer conditions are 10 mM sodium phosphate at pH 11.7. The plan is to gather data in triplicate with fresh solutions of host and guest on each occasion.  Also, all hosts samples are probed by one of either two ways to determine the waters of hydration in each sample.  This can be as high so this analysis is a requirement to avoid bad data.
```

Comment:
I don't understand what this means "to determine the waters of hydration". What are waters of hydration? Is it referring to the number of waters?

Reply:
Really you only need to review the changes here, which are highlighted. :) 

However, "waters of hydration" -- yes, number of waters.